pkgs/shells/fish/plugins/default.nix

```diff
@@ -2,6 +2,8 @@
 
 lib.makeScope newScope (self: with self; {
 
+  babelfish = callPackage ./babelfish.nix { };
```

Comment:
Since this can be used as a standalone executable, this probably belongs to the top level instead of the `fishPlugins` scope.

Reply:
Indeed; it felt a little weird as a plugin. I'll make this change.

Reply:
What would you recommend for the path of the package? `pkgs/shells/fish/babelfish.nix` or something else? `pkgs/tools/misc/` or similar?